src/EvoSC.Commands/ChatCommandManager.cs

```diff
@@ -14,13 +18,28 @@ public class ChatCommandManager : IChatCommandManager
     private readonly Dictionary<string, IChatCommand> _cmds = new();
     private readonly Dictionary<string, string> _aliasMap = new();
     private readonly Dictionary<Type, List<IChatCommand>> _controllerCommands = new();
+    private readonly IValueReaderManager _valueReader;
 
     private readonly ILogger<ChatCommandManager> _logger;
 
-
-    public ChatCommandManager(ILogger<ChatCommandManager> logger)
+    public IValueReaderManager ValueReader => _valueReader;
+    
+    public ChatCommandManager(ILogger<ChatCommandManager> logger, IPlayerManagerService playerManager)
     {
         _logger = logger;
+        _valueReader = SetupValueReader(playerManager);
+    }
+
+    private IValueReaderManager SetupValueReader(IPlayerManagerService playerManager)
```

Comment:
I would consider adding private methods below public. It's good to keep a clear hierarchy for public/private methods in classes.

Reply:
I usually put methods which are only used in the constructor close to the constructor so its all in the same place for easy navigation.